bundles/org.eclipse.core.databinding.beans/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.core.databinding.beans
-Bundle-Version: 1.8.0.qualifier
+Bundle-Version: 1.9.0.qualifier
```

Comment:
Just a thought that I find interesting:

This Eclipse way-of-working with "controlled deletions" could benefit from a 5-part version string:

1: Qualifier
2: Patch
3: Minor, backwards compatible
4: New part: Medium, only controlled, previously announced deletions
5: Major, Not backwards compatible

Just a thought.

Reply:
Yeah, but that would be again a "Eclipse" only solution which nobody outside the Eclipse world would be using or undersatnd.